install/kubernetes/helm/subcharts/galley/templates/deployment.yaml

```diff
@@ -95,6 +99,7 @@ spec:
       - name: certs
         secret:
           secretName: istio.istio-galley-service-account
+          optional: true
```

Comment:
This will likely cause the webhook to fail on startup. Galley may start before the k8s key/cert secret is propagated. This change probably needs to be conditional on configValidation=false.